### PR TITLE
Add Microsoft To Do, BGram, Oto Music, Simplenote, Yeelight, GlobeOne

### PR DESCRIPTION
**Pull request by ninopiamonte**

### CODE REVIEW

**Comment by paphonb** [comment]
Please add 4px padding on all sides of the icon to make the size consistent. Also I couldn't see the xml file, did you save it properly?

**Comment by ninopiamonte** [comment]
> Please add 4px padding on all sides of the icon to make the size consistent. Also I couldn't see the xml file, did you save it properly?

Updated. Let me know if things are working properly

**Comment by ninopiamonte** [comment]
Corrected the margin/padding to 1px per side. Sorry for that!

**Comment by ninopiamonte** [comment]
Add BGram (Telegram client)

File: svgs/microsoft_todo.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.81,11.52l.32-.32.32-.33L6.16,7.53a1.26,1.26,0,0,0-1.8,0l-3,3a1.29,1.29,0,0,0,0,1.8L4.93,16a2.07,2.07,0,0,1,.15-.5,1.69,1.69,0,0,1,.24-.41l.2-.24c.69-.71,1.38-1.4,2.08-2.1ZM22.64,6.74l-3-3.09a1.27,1.27,0,0,0-1.76,0L10.3,11.31l-.2.21h0l-.32.33h0l-.32.32-.58.59-.62.63c-.7.69-1.39,1.39-2.08,2.09a1.57,1.57,0,0,0-.19.26,1.23,1.23,0,0,0-.16.49,1.15,1.15,0,0,0,0,.32,1.36,1.36,0,0,0,0,.17s0,.08,0,.12A.54.54,0,0,1,6,17a1,1,0,0,0,.11.17l.08.09.08.09,3,3,0,0,.09.08a1.88,1.88,0,0,0,.18.12,1.23,1.23,0,0,0,1.32-.07.71.71,0,0,0,.09-.07l.07-.06,0,0,3-3L22.64,8.5A1.26,1.26,0,0,0,22.64,6.74Z"/></svg>
```
Comment (patrykmichalik) [inline svgs/microsoft_todo.svg:1]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   id="Layer_1"
   data-name="Layer 1"
   viewBox="0 0 24 24"
   version="1.1"
   sodipodi:docname="microsoft_todo.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs7">
    <inkscape:path-effect
       effect="fillet_chamfer"
       id="path-effect3665"
       is_visible="true"
       lpeversion="1"
       satellites_param="F,0,1,1,0,12,0,1 @ F,0,1,1,0,12,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 | F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1 @ F,0,0,1,0,0,0,1"
       unit="px"
       method="auto"
       mode="F"
       radius="12"
       chamfer_steps="1"
       flexible="false"
       use_knot_distance="true"
       apply_no_radius="true"
       apply_with_radius="true"
       only_selected="true"
       hide_knots="false" />
    <inkscape:path-effect
       effect="fillet_chamfer"
       id="path-effect3329"
       is_visible="true"
       lpeversion="1"
       satellites_param="F,0,0,1,0,8,0,1 @ F,0,0,1,0,8,0,1 @ F,0,0,1,0,8,0,1 @ F,0,0,1,0,8,0,1"
       unit="px"
       method="auto"
       mode="F"
       radius="8"
       chamfer_steps="1"
       flexible="false"
       use_knot_distance="true"
       apply_no_radius="true"
       apply_with_radius="true"
       only_selected="false"
       hide_knots="false" />
    <inkscape:path-effect
       effect="fillet_chamfer"
       id="path-effect3323"
       is_visible="true"
       lpeversion="1"
       satellites_param="F,0,1,1,0,8,0,1 @ F,0,0,1,0,8,0,1 @ F,0,0,1,0,8,0,1 @ F,0,0,1,0,8,0,1"
       unit="px"
       method="auto"
       mode="F"
       radius="8"
       chamfer_steps="1"
       flexible="false"
       use_knot_distance="true"
       apply_no_radius="true"
       apply_with_radius="true"
       only_selected="true"
       hide_knots="false" />
    <inkscape:path-effect
       effect="fillet_chamfer"
       id="path-effect3317"
       is_visible="true"
       lpeversion="1"
       satellites_param="F,0,0,1,0,12,0,1 @ F,0,0,1,0,12,0,1 @ F,0,0,1,0,12,0,1 @ F,0,0,1,0,12,0,1"
       unit="px"
       method="auto"
       mode="F"
       radius="12"
       chamfer_steps="1"
       flexible="false"
       use_knot_distance="true"
       apply_no_radius="true"
       apply_with_radius="true"
       only_selected="false"
       hide_knots="false" />
    <inkscape:path-effect
       effect="fillet_chamfer"
       id="path-effect2570"
       is_visible="true"
       lpeversion="1"
       satellites_param="F,0,1,1,0,8,0,1 @ F,0,0,1,0,8,0,1 @ F,0,0,1,0,8,0,1 @ F,0,0,1,0,8,0,1"
       unit="px"
       method="auto"
       mode="F"
       radius="8"
       chamfer_steps="1"
       flexible="false"
       use_knot_distance="true"
       apply_no_radius="true"
       apply_with_radius="true"
       only_selected="true"
       hide_knots="false" />
    <inkscape:path-effect
       effect="fillet_chamfer"
       id="path-effect2469"
       is_visible="true"
       lpeversion="1"
       satellites_param="F,0,0,1,0,8,0,1 @ F,0,0,1,0,8,0,1 @ F,0,0,1,0,8,0,1 @ F,0,0,1,0,8,0,1"
       unit="px"
       method="auto"
       mode="F"
       radius="8"
       chamfer_steps="1"
       flexible="false"
       use_knot_distance="true"
       apply_no_radius="true"
       apply_with_radius="true"
       only_selected="false"
       hide_knots="false" />
  </defs>
  <sodipodi:namedview
     id="namedview5"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="true"
     inkscape:zoom="17.105504"
     inkscape:cx="-9.2367928"
     inkscape:cy="15.462859"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="Layer_1">
    <inkscape:grid
       type="xygrid"
       id="grid843"
       empspacing="1" />
  </sodipodi:namedview>
  <g
     id="g3984"
     transform="matrix(0.07405793,0,0,0.07405793,31.847525,8.4822483)">
    <path
       id="path3319"
       style="fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
       d="m -361.48528,3.4852814 -38.02944,38.0294376 a 12,12 90.000001 0 0 0,16.970563 l 40.86042,40.860421 c 0.84287,-2.593074 2.20713,-5.041107 4.24024,-7.074219 l 49.34375,-49.341796 -39.44441,-39.4444068 a 12,12 180 0 0 -16.97056,2e-7 z M -290.92969,57.070312 -338.85742,105 -298,145.85742 -250.07031,97.929688 Z"
       inkscape:path-effect="#path-effect3665"
       inkscape:original-d="m -353,-5 -55,55 49.3457,49.345703 c 0.84287,-2.593074 2.20713,-5.041107 4.24024,-7.074219 l 49.34375,-49.341796 z M -290.92969,57.070312 -338.85742,105 -298,145.85742 -250.07031,97.929688 Z" />
    <path
       style="fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
       d="M -289.51472,151.51472 -136.48528,-1.5147186 a 12,12 90 0 0 0,-16.9705624 l -38.02944,-38.029438 a 12,12 180 0 0 -16.97056,0 L -344.51472,96.514719 a 12,12 90 0 0 0,16.970561 l 38.02944,38.02944 a 12,12 180 0 0 16.97056,0 z"
       id="path3313"
       inkscape:path-effect="#path-effect3317"
       inkscape:original-d="m -298,160 170,-170 -55,-55 -170,170 z" />
  </g>
</svg>
```

File: app/src/main/res/drawable/microsoft_todo.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+
+    <path
+        android:fillColor="#000000"
+        android:pathData="M8.81,11.52 L9.13,11.2 L9.45,10.87 L6.16,7.53 A1.26,1.26,0,0,0,4.36,7.53 L1.36,10.53 A1.29,1.29,0,0,0,1.36,12.33 L4.93,16 A2.07,2.07,0,0,1,5.08,15.5 A1.69,1.69,0,0,1,5.32,15.09 L5.52,14.85 C6.21,14.14,6.9,13.45,7.6,12.75 Z M22.64,6.74 L19.64,3.65 A1.27,1.27,0,0,0,17.88,3.65 L10.3,11.31 L10.1,11.52 L10.1,11.52 L9.78,11.85 L9.78,11.85 L9.46,12.17 L8.88,12.76 L8.26,13.39 C7.56,14.08,6.87,14.78,6.18,15.48 A1.57,1.57,0,0,0,5.99,15.74 A1.23,1.23,0,0,0,5.83,16.23 A1.15,1.15,0,0,0,5.83,16.55 A1.36,1.36,0,0,0,5.83,16.72 S5.83,16.8,5.83,16.84 A0.54,0.54,0,0,1,6,17 A1,1,0,0,0,6.11,17.17 L6.19,17.26 L6.27,17.35 L9.27,20.35 L9.27,20.35 L9.36,20.43 A1.88,1.88,0,0,0,9.54,20.55 A1.23,1.23,0,0,0,10.86,20.48 A0.71,0.71,0,0,0,10.95,20.41 L11.02,20.35 L11.02,20.35 L14.02,17.35 L22.64,8.5 A1.26,1.26,0,0,0,22.64,6.74 Z" />
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/microsoft_todo.xml:11]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:pathData="m5.0767,8.7404 l-2.8164,2.8164a0.8887,0.8887 60.5449,0 0,0 1.2568l3.026,3.026c0.0624,-0.192 0.1635,-0.3733 0.314,-0.5239l3.6543,-3.6542 -2.9212,-2.9212a0.8887,0.8887 81.1293,0 0,-1.2568 0zM10.3019,12.7088 L6.7524,16.2583 9.7783,19.2841 13.3278,15.7347Z"
      android:strokeLineJoin="miter"
      android:strokeWidth="1"
      android:fillColor="#000000"
      android:strokeColor="#00000000"
      android:strokeLineCap="butt"/>
  <path
      android:pathData="M10.4067,19.7031 L21.7397,8.3701a0.8887,0.8887 60.5449,0 0,0 -1.2568l-2.8164,-2.8164a0.8887,0.8887 125.4128,0 0,-1.2568 0L6.3335,15.6299a0.8887,0.8887 106.9495,0 0,0 1.2568l2.8164,2.8164a0.8887,0.8887 63.6484,0 0,1.2568 0z"
      android:strokeLineJoin="miter"
      android:strokeWidth="1"
      android:fillColor="#000000"
      android:strokeColor="#00000000"
      android:strokeLineCap="butt"/>
</vector>
```

File: svgs/oto_music.svg [hunk 3]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><circle cx="18.95" cy="9.16" r="1.2"/><path d="M17,1a6.19,6.19,0,0,0-6,4.81,6.37,6.37,0,0,0-.16,1.39v9.7a1.29,1.29,0,1,1-1.28-1.28,1.23,1.23,0,0,1,.77.26v-5a6.49,6.49,0,0,0-.77,0,6.11,6.11,0,1,0,6.09,6.36c0-.09,0-.17,0-.26V8.35a2.54,2.54,0,0,1,2.54-2.54h2.44V1Z"/><path d="M9.9,5.81A7.1,7.1,0,0,1,13.28,1H5.82V5.81Z"/></svg>
```
Comment (patrykmichalik) [inline svgs/oto_music.svg:1]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   id="Layer_1"
   data-name="Layer 1"
   viewBox="0 0 24 24"
   version="1.1"
   sodipodi:docname="oto_music.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs11" />
  <sodipodi:namedview
     id="namedview9"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     inkscape:zoom="16"
     inkscape:cx="-6.0625"
     inkscape:cy="11.71875"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="Layer_1" />
  <g
     id="g993"
     transform="translate(-2.473506e-4,-1.0309094e-4)">
    <circle
       cx="18.260134"
       cy="9.3846216"
       r="1.0859737"
       id="circle2"
       style="stroke-width:0.904978" />
    <path
       id="path4"
       style="stroke-width:0.904978"
       d="m 16.496094,2 a 5.6018146,5.6018146 0 0 0 -5.429688,4.3535156 5.7647106,5.7647106 0 0 0 -0.146484,1.2578125 v 8.7773439 a 1.1674218,1.1674218 0 1 1 -1.1582032,-1.158203 1.1131231,1.1131231 0 0 1 0.3085932,0.04687 v -4.34375 a 5.873308,5.873308 0 0 0 -0.3085932,0.0078 5.5294163,5.5294163 0 1 0 5.5117192,5.75586 V 16.460938 8.6523438 A 2.2986444,2.2986444 0 0 1 17.572266,6.3535156 H 19.78125 V 2 Z" />
    <path
       d="M 10.070081,6.3529448 A 6.4253447,6.4253447 0 0 1 13.128907,2 H 6.3777703 v 4.3529448 z"
       id="path6"
       style="stroke-width:0.904978" />
  </g>
</svg>
```

File: svgs/bgram.svg [hunk 4]
```diff
@@ -0,0 +1 @@
...13,2.43l2.46,2.59c.3-.45.74-1.13,1.09-1.76A35.16,35.16,0,0,0,11,10.25Z"/><path d="M11.74,10a35.44,35.44,0,0,1-1.81,3.82c-.18.32-.38.65-.57,1l.71-.44.46-.27a18,18,0,0,1,3.52-1.61c.49-1.18,1.07-2.66,1.39-3.69C14.31,9.05,13,9.53,11.74,10Z"/><path d="M19.55,6.93a6.63,6.63,0,0,1,2-2.06A1.57,1.57,0,0,0,20.26,4h0a4.37,4.37,0,0,0-1.91.66c-.47.23-1,.5-1.67.77a15.49,15.49,0,0,1-.43,2.53,16.57,16.57,0,0,1,2.93-.37A5.41,5.41,0,0,1,19.55,6.93ZM19.46,4.8,20,4.5A1.47,1.47,0,0,1,20,5a.48.48,0,0,1-.37.2.79.79,0,0,1-.48-.28A1.33,1.33,0,0,0,19.46,4.8Z"/><path d="M10,2.46A2.88,2.88,0,0,1,12,4a4.71,4.71,0,0,1,.48,2.15A36.37,36.37,0,0,0,16.13,5,8.21,8.21,0,0,0,16,3.27a2.64,2.64,0,0,0-.78-1.68,2.5,2.5,0,0,0-1.65-.44l-.49,0A14.07,14.07,0,0,0,8.69,2.46a5.14,5.14,0,0,1,.66,0A4,4,0,0,1,10,2.46Z"/><path d="M11.53,4.24A2.28,2.28,0,0,0,9.86,3a2.89,2.89,0,0,0-.52,0,10.5,10.5,0,0,0-3.25.76C5,4.34,2.34,6,1,6.9H2.07a57.12,57.12,0,0,0,9.87-.65A4.23,4.23,0,0,0,11.53,4.24Z"/><path d="M12,9.27c1.23-.44,2.54-.89,3.63-1.18a18,18,0,0,0,.46-2.46c-1,.37-2.19.72-3.58,1.08A8.06,8.06,0,0,1,12,9.27Z"/><path d="M17.63,11.78a21.1,21.1,0,0,1,1.24-3.63,16,16,0,0,0-2.79.41c-.26.95-.82,2.4-1.32,3.65a9.67,9.67,0,0,1,2.57-.44Z"/></svg>
```
Comment (patrykmichalik) [inline svgs/bgram.svg:1]:
```suggestion
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<svg
   id="Layer_1"
   data-name="Layer 1"
   viewBox="0 0 24 24"
   version="1.1"
   sodipodi:docname="bgram.svg"
   inkscape:version="1.1.1 (3bf5ae0d25, 2021-09-20)"
   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
   xmlns="http://www.w3.org/2000/svg"
   xmlns:svg="http://www.w3.org/2000/svg">
  <defs
     id="defs25" />
  <sodipodi:namedview
     id="namedview23"
     pagecolor="#ffffff"
     bordercolor="#666666"
     borderopacity="1.0"
     inkscape:pageshadow="2"
     inkscape:pageopacity="0.0"
     inkscape:pagecheckerboard="0"
     showgrid="false"
     showguides="true"
     inkscape:guide-bbox="true"
     inkscape:zoom="2.8615728"
     inkscape:cx="-155.15943"
     inkscape:cy="35.120547"
     inkscape:window-width="2560"
     inkscape:window-height="1015"
     inkscape:window-x="0"
     inkscape:window-y="32"
     inkscape:window-maximized="1"
     inkscape:current-layer="Layer_1" />
  <g
     id="g852"
     transform="matrix(0.90909091,0,0,0.90909091,-0.15909091,2.2133286)">
    <path
       d="m 18.06,12.41 c -0.09,0.42 -0.16,0.76 -0.19,1 -0.17,1.17 -0.48,4.6 -0.52,5.59 0,0.64 -0.3,2.57 -0.49,3.85 0.78,-1.11 2.1,-3 3.35,-5 l 0.16,-0.25 c 0.66,-1.73 1,-2.88 0.41,-3.88 -0.45,-0.72 -1.1,-1.06 -2.72,-1.31 z"
       id="path2" />
    <path
       d="M 23,10.56 C 23,9.38 22.42,8.73 21,8.32 A 5.63,5.63 0 0 0 19.51,8.13 19.5,19.5 0 0 0 18.2,11.85 c 1.55,0.25 2.47,0.6 3.07,1.64 a 2.92,2.92 0 0 1 0.35,1.89 12.63,12.63 0 0 0 1.38,-4.82 z"
       id="path4" />
    <path
       d="m 10.36,14.79 a 12.15,12.15 0 0 0 -1.5,1 l 2.37,2.48 c 0.48,-0.87 1.74,-3.23 2.25,-4.47 l 0.15,-0.36 0.14,-0.33 a 17.1,17.1 0 0 0 -2.93,1.38 z"
       id="path6" />
    <path
       d="m 11,10.25 c -1.6,0.67 -4.2,2 -5.13,2.43 l 2.46,2.59 c 0.3,-0.45 0.74,-1.13 1.09,-1.76 A 35.16,35.16 0 0 0 11,10.25 Z"
       id="path8" />
    <path
       d="m 11.74,10 a 35.44,35.44 0 0 1 -1.81,3.82 c -0.18,0.32 -0.38,0.65 -0.57,1 l 0.71,-0.44 0.46,-0.27 A 18,18 0 0 1 14.05,12.5 C 14.54,11.32 15.12,9.84 15.44,8.81 14.31,9.05 13,9.53 11.74,10 Z"
       id="path10" />
    <path
       d="m 19.55,6.93 a 6.63,6.63 0 0 1 2,-2.06 A 1.57,1.57 0 0 0 20.26,4 v 0 a 4.37,4.37 0 0 0 -1.91,0.66 c -0.47,0.23 -1,0.5 -1.67,0.77 A 15.49,15.49 0 0 1 16.25,7.96 16.57,16.57 0 0 1 19.18,7.59 5.41,5.41 0 0 1 19.55,6.93 Z M 19.46,4.8 20,4.5 A 1.47,1.47 0 0 1 20,5 0.48,0.48 0 0 1 19.63,5.2 0.79,0.79 0 0 1 19.15,4.92 1.33,1.33 0 0 0 19.46,4.8 Z"
       id="path12" />
    <path
       d="M 10,2.46 A 2.88,2.88 0 0 1 12,4 4.71,4.71 0 0 1 12.48,6.15 36.37,36.37 0 0 0 16.13,5 8.21,8.21 0 0 0 16,3.27 2.64,2.64 0 0 0 15.22,1.59 2.5,2.5 0 0 0 13.57,1.15 h -0.49 a 14.07,14.07 0 0 0 -4.39,1.31 5.14,5.14 0 0 1 0.66,0 4,4 0 0 1 0.65,0 z"
       id="path14" />
    <path
       d="M 11.53,4.24 A 2.28,2.28 0 0 0 9.86,3 2.89,2.89 0 0 0 9.34,3 10.5,10.5 0 0 0 6.09,3.76 C 5,4.34 2.34,6 1,6.9 H 2.07 A 57.12,57.12 0 0 0 11.94,6.25 4.23,4.23 0 0 0 11.53,4.24 Z"
       id="path16" />
    <path
       d="M 12,9.27 C 13.23,8.83 14.54,8.38 15.63,8.09 A 18,18 0 0 0 16.09,5.63 C 15.09,6 13.9,6.35 12.51,6.71 A 8.06,8.06 0 0 1 12,9.27 Z"
       id="path18" />
    <path
       d="m 17.63,11.78 a 21.1,21.1 0 0 1 1.24,-3.63 16,16 0 0 0 -2.79,0.41 c -0.26,0.95 -0.82,2.4 -1.32,3.65 a 9.67,9.67 0 0 1 2.57,-0.44 z"
       id="path20" />
  </g>
</svg>
```

File: app/src/main/res/drawable/oto_music.xml [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+    <path
+        android:fillColor="#000000"
+        android:pathData="M18.95,7.96 C19.6127,7.96,20.15,8.49726,20.15,9.16 C20.15,9.82274,19.6127,10.36,18.95,10.36 C18.2873,10.36,17.75,9.82274,17.75,9.16 C17.75,8.49726,18.2873,7.96,18.95,7.96 Z" />
+    <path
+        android:fillColor="#000000"
+        android:pathData="M17,1 A6.19,6.19,0,0,0,11,5.81 A6.37,6.37,0,0,0,10.84,7.2 L10.84,16.9 A1.29,1.29,0,1,1,9.56,15.62 A1.23,1.23,0,0,1,10.33,15.88 L10.33,10.88 A6.49,6.49,0,0,0,9.56,10.88 A6.11,6.11,0,1,0,15.65,17.24 C15.65,17.15,15.65,17.07,15.65,16.98 L15.65,8.35 A2.54,2.54,0,0,1,18.19,5.81 L20.63,5.81 L20.63,1 Z" />
+    <path
+        android:fillColor="#000000"
+        android:pathData="M9.9,5.81 A7.1,7.1,0,0,1,13.28,1 L5.82,1 L5.82,5.81 Z" />
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/oto_music.xml:17]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:fillColor="#FF000000"
      android:pathData="M18.2599,9.3845m-1.086,0a1.086,1.086 0,1 1,2.1719 0a1.086,1.086 0,1 1,-2.1719 0"
      android:strokeWidth="0.904978"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="m16.4958,1.9999a5.6018,5.6018 0,0 0,-5.4297 4.3535,5.7647 5.7647,0 0,0 -0.1465,1.2578v8.7773a1.1674,1.1674 0,1 1,-1.1582 -1.1582,1.1131 1.1131,0 0,1 0.3086,0.0469v-4.3438a5.8733,5.8733 0,0 0,-0.3086 0.0078,5.5294 5.5294,0 1,0 5.5117,5.7559L15.2732,16.4608 15.2732,8.6522A2.2986,2.2986 0,0 1,17.572 6.3534L19.781,6.3534L19.781,1.9999Z"
      android:strokeWidth="0.904978"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="M10.0698,6.3528A6.4253,6.4253 0,0 1,13.1287 1.9999L6.3775,1.9999v4.3529z"
      android:strokeWidth="0.904978"/>
</vector>
```

File: app/src/main/res/drawable/bgram.xml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    <path
+        android:fillColor="#000000"
+        android:pathData="M11.53,4.24 A2.28,2.28,0,0,0,9.86,3 A2.89,2.89,0,0,0,9.34,3 A10.5,10.5,0,0,0,6.09,3.76 C5,4.34,2.34,6,1,6.9 L2.07,6.9 A57.12,57.12,0,0,0,11.94,6.25 A4.23,4.23,0,0,0,11.53,4.24 Z" />
+    <path
+        android:fillColor="#000000"
+        android:pathData="M12,9.27 C13.23,8.83,14.54,8.38,15.63,8.09 A18,18,0,0,0,16.09,5.63 C15.09,6,13.9,6.35,12.51,6.71 A8.06,8.06,0,0,1,12,9.27 Z" />
+    <path
+        android:fillColor="#000000"
+        android:pathData="M17.63,11.78 A21.1,21.1,0,0,1,18.87,8.15 A16,16,0,0,0,16.08,8.56 C15.82,9.51,15.26,10.96,14.76,12.21 A9.67,9.67,0,0,1,17.33,11.77 Z" />
+</vector>
```
Comment (patrykmichalik) [inline app/src/main/res/drawable/bgram.xml:38]:
```suggestion
<vector xmlns:android="http://schemas.android.com/apk/res/android"
    android:width="24dp"
    android:height="24dp"
    android:viewportWidth="24"
    android:viewportHeight="24">
  <path
      android:fillColor="#FF000000"
      android:pathData="m16.2591,13.4951c-0.0818,0.3818 -0.1455,0.6909 -0.1727,0.9091 -0.1545,1.0636 -0.4364,4.1818 -0.4727,5.0818 0,0.5818 -0.2727,2.3364 -0.4455,3.5 0.7091,-1.0091 1.9091,-2.7273 3.0455,-4.5455l0.1455,-0.2273c0.6,-1.5727 0.9091,-2.6182 0.3727,-3.5273 -0.4091,-0.6545 -1,-0.9636 -2.4727,-1.1909z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="M20.75,11.8133C20.75,10.7406 20.2227,10.1497 18.9318,9.777A5.1182,5.1182 0,0 0,17.5773 9.6042,17.7273 17.7273,0 0,0 16.3864,12.9861c1.4091,0.2273 2.2455,0.5455 2.7909,1.4909a2.6545,2.6545 135,0 1,0.3182 1.7182,11.4818 11.4818,45 0,0 1.2545,-4.3818z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="m9.2591,15.6588a11.0455,11.0455 45,0 0,-1.3636 0.9091l2.1545,2.2545c0.4364,-0.7909 1.5818,-2.9364 2.0455,-4.0636l0.1364,-0.3273 0.1273,-0.3a15.5455,15.5455 45,0 0,-2.6636 1.2545z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="m9.8409,11.5315c-1.4545,0.6091 -3.8182,1.8182 -4.6636,2.2091l2.2364,2.3545c0.2727,-0.4091 0.6727,-1.0273 0.9909,-1.6A31.9636,31.9636 0,0 0,9.8409 11.5315Z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="m10.5136,11.3042a32.2182,32.2182 0,0 1,-1.6455 3.4727c-0.1636,0.2909 -0.3455,0.5909 -0.5182,0.9091l0.6455,-0.4 0.4182,-0.2455A16.3636,16.3636 98.12,0 1,12.6136 13.577C13.0591,12.5042 13.5864,11.1588 13.8773,10.2224 12.85,10.4406 11.6591,10.877 10.5136,11.3042Z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="m17.6136,8.5133a6.0273,6.0273 45,0 1,1.8182 -1.8727A1.4273,1.4273 0,0 0,18.2591 5.8497l0,0a3.9727,3.9727 0,0 0,-1.7364 0.6c-0.4273,0.2091 -0.9091,0.4545 -1.5182,0.7A14.0818,14.0818 135,0 1,14.6136 9.4497,15.0636 15.0636,45 0,1 17.2773,9.1133 4.9182,4.9182 0,0 1,17.6136 8.5133ZM17.5318,6.577 L18.0227,6.3042A1.3364,1.3364 0,0 1,18.0227 6.7588,0.4364 0.4364,135 0,1 17.6864,6.9406 0.7182,0.7182 0,0 1,17.25 6.6861,1.2091 1.2091,45 0,0 17.5318,6.577Z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="M8.9318,4.4497A2.6182,2.6182 0,0 1,10.75 5.8497,4.2818 4.2818,135 0,1 11.1864,7.8042 33.0636,33.0636 135,0 0,14.5045 6.7588,7.4636 7.4636,0 0,0 14.3864,5.1861 2.4,2.4 0,0 0,13.6773 3.6588,2.2727 2.2727,0 0,0 12.1773,3.2588l-0.4455,0a12.7909,12.7909 0,0 0,-3.9909 1.1909,4.6727 4.6727,0 0,1 0.6,0 3.6364,3.6364 107.6612,0 1,0.5909 0z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="M10.3227,6.0679A2.0727,2.0727 0,0 0,8.8045 4.9406,2.6273 2.6273,0 0,0 8.3318,4.9406 9.5455,9.5455 0,0 0,5.3773 5.6315C4.3864,6.1588 1.9682,7.6679 0.75,8.4861L1.7227,8.4861A51.9273,51.9273 0,0 0,10.6955 7.8951,3.8455 3.8455,135 0,0 10.3227,6.0679Z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="M10.75,10.6406C11.8682,10.2406 13.0591,9.8315 14.05,9.5679A16.3636,16.3636 0,0 0,14.4682 7.3315C13.5591,7.6679 12.4773,7.9861 11.2136,8.3133A7.3273,7.3273 135,0 1,10.75 10.6406Z"/>
  <path
      android:fillColor="#FF000000"
      android:pathData="m15.8682,12.9224a19.1818,19.1818 0,0 1,1.1273 -3.3,14.5455 14.5455,114.8725 0,0 -2.5364,0.3727c-0.2364,0.8636 -0.7455,2.1818 -1.2,3.3182a8.7909,8.7909 0,0 1,2.3364 -0.4z"/>
</vector>
```

**Review by patrykmichalik — CHANGES_REQUESTED** [review]
Thanks for contributing! Here are some tweaks I’d suggest making for consistency.

**Comment by ninopiamonte** [comment]
Thanks for the edits Patryk! Can I ask why some of the icons' corners got their weird curvy result? I'm pretty sure they were fine when I made them. I use Illustrator with this export setting Screen Shot 2021-11-07 at 11 44 09 PM

I also noticed the difference in size between your edits and mine. My artboard doc is set to 24x24px 300dpi and the icon's H/W is either 22px. Can you tell me if it's correct? Thanks :)

**Comment by patrykmichalik** [comment]
When you mention artifacts affecting the corners of some paths, are you referring to your revision of the SVGs or mine? I’ve noticed an artifact in your revision of the Microsoft To Do icon, but I’m not sure what could’ve caused it. Perhaps some kind of optimization feature is misbehaving?

Regarding your second question, your document settings are correct, but we generally ensure 2 px, not 1 px, of padding around the perimeter of the icons in Lawnicons.

**Comment by ninopiamonte** [comment]
Yes I was referring to mine. I'll check on it in Illustrator. Thanks!

**Comment by patrykmichalik** [comment]
If my suggested edits above look good to you, @ninopiamonte, could you commit them? You can do so using the buttons in the image below. You appear to have marked everything as resolved without applying the changes. Thanks in advance!

image

**Comment by ninopiamonte** [comment]
Done @patrykmichalik but it says outdated 🤔 I had no idea that that was my necessary. Thanks!

**Comment by patrykmichalik** [comment]
Suggestions are marked as outdated once the files they relate to are updated. Everything looks good to me now, so I’ll be merging this PR. Thanks a bunch for your help!

**Review by patrykmichalik — APPROVED** [review]

**Pull request merged**